docker-compose.demo.yml

```diff
@@ -2,6 +2,8 @@ version: '2'
 services:
   api:
     image: kqueen/api:v0.16
+    volumes:
```

Comment:
Why do you need to mount it in demo environment? You can take a look into containers for demo environent.

It won't provide persistent storage in any way.

Reply:
ok, btw, do we need to map logging into etcd storage?